activeadmin.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.3'
 
-  s.add_dependency 'arbre', '~> 1.1', '>= 1.1.1'
+  s.add_dependency 'arbre', '>= 1.2.0'
```

Comment:
I'm good with bumping this requirement but can we use the more conservative `~> 1.2`?

Reply:
I'm unsure whether we should put this in the changelog? I guess it's not necessary.

Reply:
I updated arbre dependency to be `~> 1.2`
I also added a line in the `unreleased` section of the changelog to mention this change

Reply:
Thanks!

@javierjulio do you think we should mention this in the change log? I think this shouldn't affect end users at all. `bundler` should be able to work it out, and arbre 1.2 over arbre 1.1 does not have any user facing changes.

Reply:
Thanks guys! @deivid-rodriguez you're right it shouldn't affect them but since Arbre is pretty important to ActiveAdmin, a mention is helpful and won't hurt so I think it's ok too. 👍🏻

Reply:
Ok!